oximeter/impl/src/schema/mod.rs

```diff
@@ -263,6 +330,45 @@ impl PartialEq for TimeseriesSchema {
 const TIMESERIES_NAME_REGEX: &str =
     "^(([a-z]+[a-z0-9]*)(_([a-z0-9]+))*):(([a-z]+[a-z0-9]*)(_([a-z0-9]+))*)$";
 
+/// Authorization scope for a timeseries.
+///
+/// This describes the level at which a user must be authorized to read data
+/// from a timeseries. For example, fleet-scoping means the data is only visible
+/// to an operator or fleet reader. Project-scoped, on the other hand, indicates
+/// that a user will see data limited to the projects on which they have read
+/// permissions.
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Deserialize,
+    Eq,
+    Hash,
+    JsonSchema,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    Serialize,
+)]
+#[serde(rename_all = "snake_case")]
+pub enum AuthzScope {
+    /// Timeseries data is limited to fleet readers.
+    Fleet,
+    /// Timeseries data is limited to the authorized silo for a user.
+    Silo,
+    /// Timeseries data is limited to the authorized projects for a user.
+    Project,
+    /// The timeseries is viewable to all without limitation.
+    None,
```

Comment:
Tiny nit - do you think a more descriptive name would be useful here? E.g. based on the comment, maybe `ViewableToAll`?

Reply:
Sure, that sounds good.

Reply:
Renamed in dca0355f3

Reply:
D'oh, actually renamed in a410171f4